### PR TITLE
Workaround introduced to detect VS and exclude the ItemGroup with the…

### PR DESCRIPTION
**Pull request by cisterni**
… copy of static files by means of

wildcards not supported by F# project system. Now you can either copy paste static or sbuild from command line
but also open the project with Visual Studio 2015.

### CODE REVIEW

File: IoX.Module.SubTree/IoX.Module.SubTree.fsproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (ranma42) [inline IoX.Module.SubTree/IoX.Module.SubTree.fsproj:1]: Unrelated to the PR?

File: IoX.Module.SubTree/IoX.Module.SubTree.fsproj [hunk 2]
```diff
@@ -35,7 +35,7 @@
   <ItemGroup>
     <Reference Include="mscorlib" />
     <Reference Include="FSharp.Core, Version=$(TargetFSharpCoreVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-      <Private>True</Private>
+      <Private>False</Private>
```
Comment (ranma42) [inline IoX.Module.SubTree/IoX.Module.SubTree.fsproj:38]: Same here

**Comment by ranma42** [comment]
Most of the changes seem unrelated to the PR (see inline comments, plus a bunch of `<Private>True</Private>` -> `<Private>False</Private>` changes)

**Comment by cisterni** [comment]
The PR also includes the removal of local copies of assemblies unneeded for Subtree and HelloWorld modules.

**Comment by cisterni** [comment]
Fixed comments in the three commits.

**Pull request merged**